test/functional/rpc_invalid_address_message.py

```diff
@@ -87,7 +84,11 @@ def test_getaddressinfo(self):
 
     def run_test(self):
         self.test_validateaddress()
-        self.test_getaddressinfo()
+
+        # skip if wallet is disabled
```

Comment:
nit: I don't think the comment is needed, when the code is self-explanatory

Reply:
Really, the comment is not necessary. Removed.